saleor/tax/calculations/__init__.py

```diff
@@ -18,3 +20,22 @@ def calculate_flat_rate_tax(
     return TaxedMoney(
         net=Money(net_amount, currency), gross=Money(gross_amount, currency)
     )
+
+
+def add_tax_to_undiscounted_price(
+    price: "Money", tax_rate: "Decimal", prices_entered_with_tax: bool
+) -> TaxedMoney:
```

Comment:
This method is almost the same as the above `calculate_flat_rate_tax`. IMO we should reuse the above one